.rubocop.yml

```diff
@@ -0,0 +1,306 @@
+Style/MethodCallParentheses:
+  Exclude:
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 2
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: snake_case, camelCase
+Style/MethodName:
+  Exclude:
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 20
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: aligned, indented
+Style/MultilineOperationIndentation:
+  Exclude:
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, MinBodyLength, SupportedStyles.
+# SupportedStyles: skip_modifier_ifs, always
+Style/Next:
+  Exclude:
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 3
+# Cop supports --auto-correct.
+Style/NilComparison:
+  Exclude:
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 38
+# Cop supports --auto-correct.
+# Configuration parameters: IncludeSemanticChanges.
+Style/NonNilCheck:
+  Exclude:
+    - 'src/clients/iscsi-lio-server.rb'
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 2
+# Configuration parameters: NamePrefix, NamePrefixBlacklist, NameWhitelist.
+# NamePrefix: is_, has_, have_
+# NamePrefixBlacklist: is_, has_, have_
+# NameWhitelist: is_a?
+Style/PredicateName:
+  Exclude:
+    - 'spec/**/*'
+    - 'src/clients/iscsi-lio-server.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 12
+# Cop supports --auto-correct.
+Style/RedundantParentheses:
+  Exclude:
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 5
+# Cop supports --auto-correct.
+# Configuration parameters: AllowMultipleReturnValues.
+Style/RedundantReturn:
+  Exclude:
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+
+# Offense count: 46
+# Cop supports --auto-correct.
+Style/RedundantSelf:
+  Exclude:
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 43
+# Cop supports --auto-correct.
+Style/SpaceAfterComma:
+  Exclude:
+    - 'src/clients/iscsi-lio-server.rb'
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 19
+# Cop supports --auto-correct.
+# Configuration parameters: AllowForAlignment.
+Style/SpaceAroundOperators:
+  Exclude:
+    - 'src/include/iscsi-lio-server/TargetData.rb'
+    - 'src/include/iscsi-lio-server/UI_dialogs.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, EnforcedStyleForEmptyBraces, SupportedStyles.
+# SupportedStyles: space, no_space, compact
+Style/SpaceInsideHashLiteralBraces:
+  Exclude:
+    - 'test/target_data_test.rb'
```

Comment:
Could we rather fix it instead of disabling in the config? There are just two offenses and it looks that this is a new file (not  an YCP legacy one), so it should be rather fixed.

Reply:
Removed exception.